### ISSUE TITLE
Unable to find a specification for SentrySDK

### ISSUE THREAD

**Issue by kenobi91** [getsentry/sentry-react-native#918]
**OS:**
- [ ]  Windows
- [x]  MacOS
- [ ]  Linux

**Platform:**
- [x]  iOS
- [ ]  Android

**SDK:**
- [x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 0.0.0

**`react-native` version:** 0.6.2

**Are you using Expo?**
- [ ]  Yes
- [x]  No

**Are you using sentry.io or on-premise?**
- [ ]  sentry.io (SaaS)
- [ ]  on-premise

**If you are using sentry.io, please post a link to your issue so we can take a look:**

[Link to issue]

**Configuration:**

(`@sentry/react-native`)
```
Sentry.init({
  dsn: 'https://...@sentry.io/...'
  // other options
});
```

or

(`react-native-sentry`)
```
Sentry.config(
  'https://...@sentry.io/...'
  // other options
 ).install();
```

---
**I have following issue:**
I'm trzing to install sentry on RN app but each time i run pod install inside my ios folder i get error:

Unable to find a specification for `SentrySDK (~> 5.1.3)` depended upon by `RNSentry`

You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.
[Description]

**Steps to reproduce:**
- Step 1
- Step 2

**Actual result:**

[Actual result]

**Expected result:**

[Expected result]

**Comment by reneeichhorn** [comment]
Having the same issue, seems to be related to the latest update `1.4.3` for now just stick to `1.4.2`
```
yarn add @sentry/react-native@1.4.2
```

**Comment by kenobi91** [comment]
Cool thx @reneeichhorn :)

**Comment by HazAT** [comment]
Sorry this is fixed in `1.4.4`
